code/modules/clothing/suits/marine_armor.dm

```diff
@@ -132,6 +132,22 @@
 	icon_state = "6"
 	armor = list("melee" = 30, "bullet" = 70, "laser" = 45, "energy" = 30, "bomb" = 60, "bio" = 10, "rad" = 10, "fire" = 30, "acid" = 30)
 
+/obj/item/clothing/suit/storage/marine/M3UL
+	name = "\improper M3-UL pattern marine armor"
+	desc = "A standard Marine M3 Ultralite Pattern Chestplate. Even lesser encumbrance and protection and cannot carry heavy weapons, but comes with a inbuilt motion detector. Provides high protection against acids and allows to be quickly taken on and off. Just be careful of machetes and bombs. Feels like nothing is here."
+	icon_state = "10"
+	armor = list("melee" = 15, "bullet" = 10, "laser" = 35, "energy" = 10, "bomb" = 5, "bio" = 0, "rad" = 0, "fire" = 5, "acid" = 50)
```

Comment:
```suggestion
	armor = list("melee" = 15, "bullet" = 10, "laser" = 35, "energy" = 10, "bomb" = 5, "bio" = 0, "rad" = 0, "fire" = 5, "acid" = 25)
```

Reply:
Wait, isn't even LESS than heavy armor?

Reply:
Yes, it's supposed to be less than heavy armor, considering the fact this is an ultra-light almost no slowdown armor.

Reply:
... You do have a point, to be honest.